apps/blog/views/post.html

```diff
@@ -2,7 +2,9 @@
 
 <div class="blog-post{% if full %} blog-post-full{% end %} block">
 {% if ! full %}
-<h3><a href="{{ url }}">{{ title }}</a></h3>
+<h3><a href="{{ fullurl }}">{{ title }}</a></h3>
+{% else %}
+<h3>{{ title }}</h3>
```

Comment:
Won't this `else` show the post title twice? We set `$page->title` to the post's title already in the `blog/post` handler.